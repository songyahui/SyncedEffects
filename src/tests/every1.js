"use hopscript"

var hh = require("hiphop");

var prg =
    <hh.module I=${{accessibility: hh.IN}} O>
      <hh.every I>
	<hh.emit O/>
      </hh.every>
    </hh.module>
;

exports.prg = new hh.ReactiveMachine(prg, "every1");
