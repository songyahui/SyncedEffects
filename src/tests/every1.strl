module every1:

input I;
output O;

every I do
      emit O;
end every

end module