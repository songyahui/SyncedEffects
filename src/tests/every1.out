every1> I;
--- Output:
every1> ;
--- Output:
every1> ;
--- Output:
every1> I;
--- Output: O
every1> I;
--- Output: O
every1> ;
--- Output:
every1> ;
--- Output:
